v0.0.10 only contains documentation

## Current Behavior
After installing the npm version TypeScript fails to find type declarations:
```
 error TS2307: Cannot find module 'emotion-rgba' or its corresponding type declarations.
```
This did not happen when testing my changes locally :(

Actually: `.\node_modules\emotion-rgba\` looks very empty:
![image](https://user-images.githubusercontent.com/1249745/168251232-dad4a81e-0beb-43d2-9e60-58f2056f7fdd.png)


## Possible Solution
I can reproduce a similar problem by running "npm pack" without running "npm run build" first.

--- a/package.json
+++ b/package.json
@@ -9,7 +9,8 @@
   "main": "dist/index.js",
   "scripts": {
     "build": "tsc",
-    "test": "jest --verbose"
+    "test": "jest --verbose",
+    "prepack": "npm ci && npm run build && npm run test"
   },
   "keywords": [
     "Emotion",

